docs/pages/eas-update/publish.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: How to publish an update
+sidebar_title: Publish an update
+description: Learn how to publish an update to a specific branch with EAS Update.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { LayersTwo02Icon } from '@expo/styleguide-icons';
+
+Publishing an update allows fixing bugs and quickly updating non-native parts of a project instead of creating a new build. This guide explains the steps required to publish an update, briefly describes how it works, and offers methods to test it.
+
+## Publish an update
+
+To publish an update with changes from your project, you can use the `eas update` command, and specify a name for the `branch` and a `message` to describe the update:
+
+<Terminal cmd={['$ eas update --branch [branch-name] --message "[message]"']} />
+
+### How does publishing an update work
+
+When you publish an update with the `eas update` command, it generates a new update bundle and uploads it to the EAS servers. The `branch` name is used to identify the update and is associated with the uploaded bundle. It is similar to how Git commit works, where every commit is associated with a Git branch.
+
+For example, you want to publish an update for the [`development` build profile](/build/eas-json/#build-profiles). You can use it as the branch name to associate the update with its build profile. When the app requests an update, it will download the update associated with the `development` branch.
+
+For this example, the command to publish an update with the `development` branch and a message is:
+
+<Terminal cmd={['$ eas update --branch development --message "Fixes typo"']} />
+
+The above command runs `npx expo export` behind the scenes to generate a **dist** directory and create a local update bundle. This update bundle is uploaded to EAS servers.
+
+<BoxLink
+  title="In-depth guide on how EAS Update works"
+  description="Dive deep and learn how EAS Update works."
+  href="/eas-update/how-it-works/"
+  Icon={LayersTwo02Icon}
+/>
+
+## Test a published update
```

Comment:
To add on to what brent said, it would be nice to include instructions to view the update in a development client build, where you can view the published update in the extensions tab and open it

Reply:
Thanks for the suggestion!

I had a task to update the "Use expo-dev-client with EAS Update" guide and add/cover info about using **Extensions** tab to view a published update using the **Extensions** tab. Do you think it is okay to link it under this section (Test methods)?

Here is the preview from the PR of the updated "Use expo-dev-client with EAS Update" guide: http://docs.expo.dev-pr-27772.s3-website-us-east-1.amazonaws.com/eas-update/expo-dev-client/

Otherwise, I can add those steps here, and mention the other steps as alternatives in the this guide but it might not make sense to briefly touch upon Extensions tab in "Use expo-dev-client with EAS Update" guide as it does currently because it will be duplicated information. The only unique thing that left in "Use expo-dev-client with EAS Update", if we prefer to go this route, will be "constructing a specific URL" and covering the steps to run it inside the development build.